GVFS/GVFS.Common/ProductUpgrader.cs

```diff
@@ -350,24 +353,30 @@ private bool TryRunInstallerForAsset(string name, out int exitCode, out string e
             return false;
         }
 
-        private bool TryGetLocalInstallerPath(string name, out string path)
+        private bool TryGetLocalInstallerPath(string name, out string path, out string args)
         {
-            path = null;
-
             foreach (Asset asset in this.newestRelease.Assets)
             {
                 string extension = Path.GetExtension(asset.Name);
                 if (extension != null && extension == ".exe")
                 {
-                    if ((name == GitAssetNamePrefix && asset.Name.StartsWith(GitInstallerFileNamePrefix)) ||
-                        (name == GVFSAssetNamePrefix && asset.Name.StartsWith(GVFSInstallerFileNamePrefix)))
+                    path = asset.LocalPath;
+                    if (name == GitAssetNamePrefix && asset.Name.StartsWith(GitInstallerFileNamePrefix))

```

Comment:
Not new to this PR, but you might want to use the version of `StartsWith` that takes a `StringComparison` so that we can ignore case when comparing (I doubt the case will change, but it might be a bit more robust, let me know what you think).

Reply:
(Same comment for the .exe check)